apis/sgv2-quarkus-common/src/test/java/io/stargate/sgv2/common/testresource/StargateTestResource.java

```diff
@@ -81,6 +81,9 @@ interface Defaults {
     String CLUSTER_VERSION = "4.0";
 
     String CLUSTER_DSE = null;
+
+    long CASSANDRA_STARTUP_TIMEOUT = 2;
```

Comment:
These are in minutes? Would make sense to add a comment mentioning this?

Reply:
Actually, thinking about this some more: should we require it in minutes, or would it make more sense to use seconds value? To allow more granular configurability (90 seconds etc)

Reply:
This is timeout value for starting up the service, so not sure if we need seconds level granularity. Added comments to for the defaults.